README.md

```diff
@@ -115,8 +115,8 @@ Mapbox Navigation gives you all the components you need, should you want to buil
 * [MapboxDirections](https://github.com/mapbox/MapboxDirections.swift)
   * Automotive, cycling, and walking directions
   * Traffic-influenced driving directions
-* Mapbox Navigation Core Components
-  * Route Controller
+* Mapbox Navigation core components (`MapboxNavigation` module)
```

Comment:
This and other references to modules will change once we rename `MapboxNavigation` to `MapboxCoreNavigation` and `MapboxNavigationUI` to `MapboxNavigation`.